src/projects/guides.rs

```diff
@@ -15,7 +15,7 @@ pub fn run(dir: &Path, dry_run: bool) {
     let version = Version::parse(&stdout_result[1..stdout_result.len()].trim()).unwrap();
 
     if  !req.matches(&version) {
-        panic!("Guides can only be installed with node version 16 right now. you have {:?}", stdout_result)
+        panic!("Guides can only be installed with node version 14 right now. you have {:?}", stdout_result)
```

Comment:
a) nice catch 😂 
b) is this true any more? now that we've released the redesign we don't use sass and we probably don't have any restrictions on Node version any more 🤔

Reply:
I think you're right. It's probably not accurate. Maybe we bump to v14 and above for the moment.